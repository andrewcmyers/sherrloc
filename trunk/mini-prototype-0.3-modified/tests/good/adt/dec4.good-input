type list : * -> * = 
     Nil : forall a. list a 
   | Cons : forall a. a -> list a -> list a
