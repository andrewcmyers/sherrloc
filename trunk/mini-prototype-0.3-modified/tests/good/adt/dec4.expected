val Cons: forall a. a -> list a -> list a
val Nil: forall a. list a

